suite-native/formatters/package.json

```diff
@@ -14,9 +14,12 @@
         "@suite-common/formatters": "workspace:*",
         "@suite-common/wallet-config": "workspace:*",
         "@suite-common/wallet-core": "workspace:*",
+        "@suite-common/wallet-types": "workspace:*",
```

Comment:
Are these newly added dependencies used anywhere?

Reply:
Some of them were used and forgot to remove them. They're removed now after rebase. Altough I squashed the commits so it's easier to resolve conflicts you'll have to verify in the file.